src/org/pahappa/systems/exceptions/ValidationFailedException.java

```diff
@@ -1,8 +1,9 @@
 package org.pahappa.systems.exceptions;
 public class ValidationFailedException extends Exception {
 
-	/**
-	 * 
-	 */
+
 	private static final long serialVersionUID = 1L;
+	ValidationFailedException(String s){
```

Comment:
Maybe the parameter name can be more descriptive such as (String message)

Reply:
The same applies to this